From 32bd77c82fe12fbfe0a1d50f64fa1959cf699bea Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 3 Dec 2012 22:30:04 -0500
Subject: [PATCH] fix implicit decl warnings

Include the proper headers to fix these warnings:
libnfs-sync.c:1529:3: warning: implicit declaration of function 'gettimeofday' [-Wimplicit-function-declaration]
libnfs-zdr.c:506:2: warning: implicit declaration of function 'getuid' [-Wimplicit-function-declaration]

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 lib/libnfs-sync.c | 1 +
 lib/libnfs-zdr.c  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/lib/libnfs-sync.c b/lib/libnfs-sync.c
index fd6f796..1fbeb8f 100644
--- a/lib/libnfs-sync.c
+++ b/lib/libnfs-sync.c
@@ -45,6 +45,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <assert.h>
+#include <sys/time.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
diff --git a/lib/libnfs-zdr.c b/lib/libnfs-zdr.c
index a243d1d..3d5fd30 100644
--- a/lib/libnfs-zdr.c
+++ b/lib/libnfs-zdr.c
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
 #include "libnfs-zdr.h"
 
 struct opaque_auth _null_auth;
-- 
1.7.12.4

